### PR TITLE
Add deaccession CSV import docs, fixes issue #92

### PR DESCRIPTION
**Pull request by fiver-watson**
This pull request adds new documentation for the command-line task to import Deaccession records in CSV format. Links to the new documentation have also been added to the Deaccessions page in the User manual, and outdated links have been fixed.

### CODE REVIEW

File: admin-manual/maintenance/cli-import-export.rst [hunk 1]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+
+The expected CSV will have 7 columns, corresponding to various fields
+available in the Deaccession record template. These include:
+
+* **accessionNumber**: expects the accession number of an existing accession
+  record in AtoM as input. If no match is found for an existing accession, the
+  console will provide a warning, the row will be skipped, and the task will
+  continue.
+* **deaccessionNumber**: an identifier for the deaccession. Free text, will
+  support symbols and typographical marks such as dashes and slashes
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: add period at end

File: admin-manual/maintenance/cli-import-export.rst [hunk 2]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+* **date**: expects a date value in ISO-8601 format (YYYY-MM-DD)
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: add period

File: admin-manual/maintenance/cli-import-export.rst [hunk 3]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+* **scope**: expects one of the controlled terms from the "Scope" field in the
+  AtoM deaccession record template. English options include "Whole" and "Part"
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: add period

File: admin-manual/maintenance/cli-import-export.rst [hunk 4]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+* **description**: Free-text. Identify what materials are being deaccessioned.
+* **extent**: Free-text. Identify the number of units and the unit of
+  measurement for the amount of records being deaccessioned.
+* **reason**: Free-text. Provide a reason why the records are being
+  deaccessioned
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: add period

File: admin-manual/maintenance/cli-import-export.rst [hunk 5]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+* **culture**: Expects a 2-letter ISO 639-1 language code as input (e.g.: en,
+  fr, es, pt, etc)
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: add period

File: admin-manual/maintenance/cli-import-export.rst [hunk 6]
```diff
@@ -1562,6 +1563,85 @@ information on indexing options, see: :ref:`maintenance-populate-search-index`.
...
+There are also a number of options available with this command-line task.
+
+.. image:: images/csv-deaccession-options.*
+   :align: center
+   :width: 85%
+   :alt: An image of the command-line options for deaccession record imports
+
+By typing ``php symfony help csv:deaccession-import`` into the command-line from
+your root directory, **without** specifying the location of a CSV, you will
+able able to see the CSV import options available (pictured above). A brief
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: be able

**Review by corinnemrogers — CHANGES_REQUESTED** [review]
A few picky edits - I am conflicted about using periods in bulleted lists, but whether they are used or not should be consistent!

**Review by currmie — COMMENTED** [review]
Content looks good!

**Comment by fiver-watson** [comment]
Implemented some changes as requested, squashed and rebased!

**Review by corinnemrogers — APPROVED** [review]

**Pull request merged**